### ISSUE TITLE
Certificate details not show, certificate cannot be removed: HC_CBOR_DECODING_FAILED

### ISSUE THREAD

**Issue by max-horvath** [corona-warn-app/cwa-app-ios#4210]
## Avoid duplicates
- [x] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [x] Bug is specific for iOS only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [x] Bug is not already reported in another issue

## Technical details

- Device name: iPhone 12 Pro Max
- iOS version: 15.3.1
- App version: 2.17.1

## Describe the bug

A certificate has been added to my CWA quite a while back.

Recently it started to show:

```
Optional(HealthCertificateToolkit.CertificateDecodingError. HC_CBOR_DECODING_FAILED(nil))
```
While I don't care too much about this error (the certificate is older and unused anyway), I'm unable to remove it from the device. Nothing happens when clicking on the QR code, no screen with details pushed, so I cannot delete the certificate.

## Steps to reproduce the issue

I honestly have no idea.

## Expected behaviour

Deleting an invalid certificate should be possible.

## Possible Fix

Show no details about the certificate in the details view (since decoding failed), but let me delete it from the app.

---
Internal Tracking ID: EXPOSUREAPP-11802

**Comment by thomasaugsten** [comment]
I removed the QR code. We will check why the qr code has a decoding error. 
We hope to resolve the error as soon as possible

**Comment by Ein-Tim** [comment]
@max-horvath 

Actually, your report has not been changed at all but @thomasaugsten only removed the QR code. All other things (your description, etc.) are still the same.

**Comment by thomasaugsten** [comment]
The issue is the wrong format of the iat it should be integer instead of string

**Comment by Ein-Tim** [comment]
PR: https://github.com/corona-warn-app/cwa-app-ios/pull/4230

**Comment by Ein-Tim** [comment]
@thomasaugsten 

In the description of https://github.com/corona-warn-app/cwa-app-ios/pull/4230 it says that you want users to contact the team if they encounter such an error, but you are not showing e.g. a pop up to the user "Decoding of the certificate failed. Please contact the technical hotline."

Wouldn't this make sense?

**Comment by Ein-Tim** [comment]
@max-horvath 

Can you check whether this is fixed in version 2.18? You'll have to tap 7 times on the certificate if you want to delete it.

**Comment by max-horvath** [comment]
> @thomasaugsten 
> 
> In the description of https://github.com/corona-warn-app/cwa-app-ios/pull/4230 it says that you want users to contact the team if they encounter such an error, but you are not showing e.g. a pop up to the user "Decoding of the certificate failed. Please contact the technical hotline."
> 
> Wouldn't this make sense for you and help the user? 

I have to agree that the current implementation is not intuitive at all. I would be surprised if many end users would detect this hint to reach out.

I was only able to verify the implementation because you described it. I would have never found this work-around on my own.

**Comment by max-horvath** [comment]
> @max-horvath 
> 
> Can you check whether this is fixed in version 2.18? You'll have to tap 7 times on the certificate if you want to delete it. 

The functionality works according to the implementation, but I would hope you would reconsider improving it.
